test/example/deployment.yaml

```diff
@@ -19,7 +33,7 @@ spec:
     spec:
       containers:
       - name: example
-        image: docker.io/library/busybox:stable
+        image: docker.io/library/nginx:stable
```

Comment:
Why we replace busybox with nginx? The commit message can explain what we are
trying to do.

Reply:
Updated new chnage set